File: client/client/src/app/Model/Product.ts
export interface Product {
  id: number;
  categoryId: number;
  name: string;
  image: string;  // URL לתמונה או אמוג'י
}

export interface SelectedProduct {
  product: Product;
  quantity: number;
}

export interface ProductDTO {
  name: string;
  image: string;
  categoryId: number;
}

File: Models/Product.cs

    public class Product
    {
        public int Id { get; set; }
        public int CategoryId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Image { get; set; } = string.Empty;
        public virtual Category? Category { get; set; }

    }
